extensions/amp-lightbox/0.1/amp-lightbox.js

```diff
@@ -189,6 +189,7 @@ class AmpLightbox extends AMP.BaseElement {
     });
 
     if (this.isScrollable_) {
+      this.element./*OK*/scrollTop = this.element./*OK*/scrollTop || 1;
```

Comment:
So you mean calling `this.scrollHandler_()` inside `update_() `? If yes, can you explain why the the lightbox is not yet visible inside `initialize_()` but in ` in update_()`? I guess the answers are straightforward but I didn't yet fully understand with the lifecycle of amp elements, so sorry for the tons of questions :)